lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -354,6 +354,7 @@ <h4 class="message-title">${_('Your verification will expire soon!')}</h4>
                     <a class="action action-upgrade" href="${ecommerce_payment_page}?sku=${course_mode_info['verified_sku']}">
                   % else:
                     <a class="action action-upgrade" href="${reverse('verify_student_upgrade_and_verify', kwargs={'course_id': unicode(course_overview.id)})}" data-course-id="${course_overview.id}" data-user="${user.username}">
+                    <span class="sr">&nbsp;${course_overview.display_name_with_default}</span>
```

Comment:
please add it [here](https://github.com/edx/edx-platform/blob/attiya/ECOM-4269-and-4270/lms/templates/dashboard/_dashboard_course_listing.html#L361) as we would need this to be present with both ecommerce and simple payment flow methods. Also add the translation.